refresh/index.js

```diff
@@ -48,7 +114,90 @@ module.exports = generators.Base.extend({
       }
     },
 
+    createBasicProject: function() {
```

Comment:
Hold on, that might not be a good idea bearing in mind **these** writes need to happen before `readModels` with the way things are currently laid out.